### DISCUSSION TITLE
What about EA Frameworks in an Antifragile World

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#32] [Ideas]
When the third of the three different schools of Enterprise Architecture, Enterprise Ecological
Adaption (Lapalme, 2012), is more suitable for an antifragile enterprise architecture
and the residuality theory of B. O’Reilly (2021, p. 809) is advocating Post-Structuralism,
then what is still the added value for frameworks like TOGAF, FEAF, ....

Just something to think about and let it sink in.

**Comment by edzob** [comment]
AF needs a strong body (Seneca's Barbbell / hydra's body)
The world is a system of systems (Senge & Cynefone) 
Diversity is a thing of reality and needed (CAS Resilience & AF)

:)

**Reply by JRBliekendaal** [reply]
So the essential part. The part that should stay (the residu) that is important should be based on an intentional design/architecture. The rest should be emergent, as small as possible, as loosely coupled as possible. The core should be the stability but as small as possible. This is btw exactly what O'Reilly means to say with his philosophy. Architecture is most of the time about building blocks and patterns. Most of them are black boxes. This is the paradigm shift in enterprise architecture. You do not want a black box. You need the white box to determine the residu. 

All things are coming together. I think I might have something here what could steer me into the direction of the succesfactors of EA in a antifragile world.
